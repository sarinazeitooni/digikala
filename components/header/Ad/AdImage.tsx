import React from 'react';
import style from './scss/ad-image.module.scss';
interface props {
    image : string
}
const AdImage = ({image} : props)=>{
    return(
        <div className={style['ad-image-container']}>
            <img style={{background : `url("${image}")` }} className={style['ad-image']}/>
        </div>
    )
};
export default AdImage;