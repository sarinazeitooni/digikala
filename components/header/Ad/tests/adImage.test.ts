import * as React from 'react';
import {shallow} from "enzyme";
import AdImage from "../AdImage";
describe("ad image component",()=>{
    it("should match snap shot" , ()=>{
        const wrapper = shallow(<AdImage/);
        expect(wrapper).toMatchSnapshot();
    });
    it("should render image correctly" , ()=>{
        const wrapper = shallow(<AdImage/>);
        const container = wrapper.find("div.ad-image-container");
        expect(container.exists()).toBe(true);
    });
});