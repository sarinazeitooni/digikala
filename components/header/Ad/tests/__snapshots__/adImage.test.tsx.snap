// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ad image component should match snap shot 1`] = `ShallowWrapper {}`;
